### ISSUE TITLE
How to apply a different tonemapping algorithm?

### ISSUE THREAD

**Issue by Harper714** [google-research/multinerf#31]
Hi, if I want to get the result of HDR tonemapping, I can change exposure and the parameters in linear_to_srgb, right? 
Is there any advice to change these parameters?

**Comment by jonbarron** [comment]
Yeah I guess you could directly change that function to have a different shape, but it's used during training so I'm not sure what effect that will have. The easiest thing to do is to train a rawnerf, render a linear intensity image, and then apply your own tone mapping function to the linear rendering that you get out.

**Comment by Harper714** [comment]
Thanks for the answer.

BTW, I trained the model for the scene parkstatue using default configs, and rendered the views by setting Config.factor =1. Then I noticed that when zoom-in the rendered image, i.e., color_000.png, there are obvious stripe artifacts on the sky region near the lamp. Have you noticed this issue, and what do you think the reason is?
